pkgs/tools/graphics/vulkan-cts/default.nix

```diff
@@ -104,6 +106,10 @@ stdenv.mkDerivation (finalAttrs: {
   '';
 
   passthru.updateScript = ./update.sh;
+  passthru.tests.lavapipe = runCommand "vulkan-cts-tests-lavapipe" { nativeBuildInputs = [ finalAttrs.finalPackage ]; } ''
+    VK_ICD_FILENAMES=${mesa.drivers}/share/vulkan/icd.d/lvp_icd.x86_64.json deqp-vk -n dEQP-VK.api.smoke.triangle
```

Comment:
Oh wait, is hardcoding 86_64 intended!

Reply:
Now using a `*` with some weird bash because `export` does not expand wildcards.